### DISCUSSION TITLE
Monospace font in mobile code viewer

### DISCUSSION THREAD

**Discussion by tomiolah** [community/community#5786] [Mobile] [answered]
It's really weird for the code preview to be presented with a non-monospace font (it's probably the system font).
I'm not saying to support custom fonts (although that would be pretty sweat), I would just like to see the code in a more readable, code-friendly monospace font (the one used on the web version would be just fine).

Thanks in advance! 😃

**Comment by mxie** [comment, marked as answer]
Thanks for your feedback @tomiolah! Would you mind sharing some screenshots of what you're seeing?

**Reply by tomiolah** [reply]
@mxie Yeah, sure!
Currently the GitHub Mobile code viewer on my OnePlus Nord (Android 11) device looks like this when opening for example a TypeScript file:
![Screenshot_20211102-235216__01.jpg](https://user-images.githubusercontent.com/482524/139957108-861c8824-a1b6-400a-a3db-8f9bea4593c7.jpg)

It would be nice to have a monospaced font set for the code instead of the display font.

**Reply by rjaletzki** [reply]
Same for me (Android 11 on OnePlus Nord2 5G). The `Code Options` settings already allow for overriding the system font size, I would hope for a monospace option - or defaulting to monospace like in the web view.
![code-options](https://user-images.githubusercontent.com/16937760/140100220-d8011b2d-13d0-47ab-874b-c2649701dbd7.jpg)

**Reply by mxie** [reply]
Thanks for sharing your screenshots, @tomiolah and @rjaletzki! The app is likely using whatever system font you had chosen, which isn't ideal. 😬 I've  gone ahead and relayed this issue back to the team. We'll get back to you with any updates we have!
